libs/martech/attributes.js

```diff
@@ -73,9 +73,14 @@ export async function decorateSectionAnalytics(section, idx, config) {
   });
   const mepMartech = config?.mep?.martech || '';
   section.querySelectorAll('[data-block]').forEach((block, blockIdx) => {
-    const blockName = block.classList[0] || '';
-    block.setAttribute('daa-lh', `b${blockIdx + 1}|${blockName.slice(0, 15)}${mepMartech}`);
-    decorateDefaultLinkAnalytics(block, config);
+    const lhAtt = block.getAttribute('daa-lh');
+    if (!lhAtt) {
+      const blockName = block.classList[0] || '';
+      block.setAttribute('daa-lh', `b${blockIdx + 1}|${blockName.slice(0, 15)}${mepMartech}`);
+      decorateDefaultLinkAnalytics(block, config);
+    } else {
+      block.setAttribute('daa-lh', `${lhAtt}${mepMartech}`);
+    }
```

Comment:
```suggestion
    let lhAtt = block.getAttribute('daa-lh');
    if (!lhAtt) {
      const blockName = block.classList[0] || '';
      lhAtt = `b${blockIdx + 1}|${blockName.slice(0, 15)}`;
    }
    block.setAttribute('daa-lh', `${lhAtt}${mepMartech}`);
    decorateDefaultLinkAnalytics(block, config);
```
Need to keep decorateDefaultLinkAnalytics outside of the if statement.  And I think this suggestion is cleaner.